smart-contracts/contracts/lp-strategy/src/execute.rs

```diff
@@ -1027,28 +1008,19 @@ mod tests {
 
         let res = handle_icq_ack(deps.as_mut().storage, env, to_binary(&ibc_ack).unwrap()).unwrap();
 
-        // get the failed pending bonds total amount
-        let failed_total_amount = failed.bonds.iter().fold(Uint128::zero(), |acc, bond| {
-            let amount = match bond.raw_amount {
-                RawAmount::LocalDenom(amount) => amount,
-                RawAmount::LpShares(_) => panic!("unexpected lp shares"),
-            };
-            acc + amount
-        });
-
         // get the pending bonds total amount
         let pending_total_amount = pedning_bonds
             .iter()
             .fold(Uint128::zero(), |acc, bond| acc + bond.amount);
 
-        // check that the res amount matches the amount in both queues
+        // check that the res amount matches the amount in the pending queue ONLY
```

Comment:
We should also verify that the failed bond queue is properly emptied

Reply:
we already do this a few lines below:
```
        // check that the failed join & pending queues are emptied
        assert!(FAILED_JOIN_QUEUE.is_empty(&deps.storage).unwrap());
        assert!(PENDING_BOND_QUEUE.is_empty(&deps.storage).unwrap());
```